### PR TITLE
Unescaping HTML in WordPress tags and categories.

### LINKED ISSUE #2557: Wordpress Importer: Tags: &amp; should be just &
I noticed that imports made through the wordpress importer plugin does not correctly handle ampersands (*&*) in tags and writes them into the new site as &amp;.

This probably applies to other HTML entities aswell.

### PR DESCRIPTION
**Pull request by felixfontein**
Fixes #2557.

@okin: can you test this with your older export?

### CODE REVIEW

**Comment by okin** [comment]
Worked for me with Python 3.5 and code looks sane to me aswell.

Thank you @felixfontein!

**Comment by felixfontein** [comment]
Good to hear! I'd like @Kwpolska to also take a look, especially at the function in `utils`, before we merge.

File: nikola/utils.py [hunk 1]
```diff
@@ -1942,3 +1942,20 @@ def load_data(path):
         return
     with io.open(path, 'r', encoding='utf8') as inf:
         return loader.load(inf)
+
+
+# http://stackoverflow.com/a/2087433
+if sys.version_info[0] == 3 and sys.version_info[1] >= 4:
```
Comment (Kwpolska) [inline nikola/utils.py]: Breaks with Python 4.0.

I’d suggest:

```py
try:
    import html
    html_unescape = html.unescape
except (AttributeError, ImportError):
    try:
        # the whole ordeal
```

Or even don’t use html.unescape and always define our own.

Comment (felixfontein) [inline nikola/utils.py]: Yeah, that sounds better. About defining my own: doesn't sound like a very appealing idea to me.

**Pull request merged**